lambda_function.py

```diff
@@ -3,6 +3,7 @@
 import json
 import base64
 import boto3
+import requests
```

Comment:
Please update requests.txt to include this package

Reply:
ah, good catch! updated